q2cli/builtin/tools.py

```diff
@@ -767,13 +876,11 @@ def cache_status(cache):
             for key in _cache.get_keys():
                 key_values = _cache.read_key(key)
 
-                if 'data' in key_values:
-                    data = key_values['data']
+                if (data := key_values['data']) is not None:
                     data_output.append(
                         'data: %s -> %s' %
                         (key, str(Result.peek(_cache.data / data))))
-                elif 'pool' in key_values:
-                    pool = key_values['pool']
+                elif (pool := key_values['pool']) is not None:
```

Comment:
@colinvwood Looks like the issue is right here probably from the merge. We want the version where it checks `if 'data' in key_values` and the same for 'pool'

Reply:
Alternatively, if we want to be cool

```
                if (data := key_values.get('data', None)) is not None:
                    data_output.append(
                        'data: %s -> %s' %
                        (key, str(Result.peek(_cache.data / data))))
                elif (pool := key_values.get('pool', None)) is not None:
                    pool_output.append(
                        'pool: %s -> size = %s' %
                        (key, str(len(os.listdir(_cache.pools / pool)))))
 ```